packages/components/card/stories/card.happo.js

```diff
@@ -0,0 +1,65 @@
+import { storiesOf, html } from '@open-wc/demoing-storybook';
+import '@tradeshift/elements';
+import '@tradeshift/elements.card';
+
+import { sizes, orientations, types } from '../src/utils';
+
+storiesOf('ts-card', module)
+	.add('Simple', () => {
+		const properties = { types, sizes, orientations };
+		const propKeys = Object.keys(properties);
+
+		const templates = [];
+		const content =
+			'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam dolorem ex iste laborum perspiciatis. Impedit ipsa velit voluptatum. A cum, dolore fugiat iure magni nam natus odio quibusdam quod recusandae.';
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 214 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3ALineLength)